lib/iris/fileformats/_pyke_rules/fc_rules_cf.krb

```diff
@@ -268,13 +268,14 @@ fc_build_auxiliary_coordinate_latitude
     assert
         python cf_coord_var = engine.cf_var.cf_group.auxiliary_coordinates[$coordinate]
         python build_auxiliary_coordinate(engine, cf_coord_var,
-                                          coord_name=CF_VALUE_STD_NAME_LAT)
+                                          coord_name=CF_VALUE_STD_NAME_LAT,
+                                          coord_system=engine.provides['coordinate_system'])
```

Comment:
How does the *correct* coordinate system find its way into the coordinate? I'd be concerned that we could get the one for the dimension coordinates...